SimDataFormats/TrackingAnalysis/src/classes_def.xml

```diff
@@ -40,4 +40,8 @@
   <class name="edm::Wrapper<std::unordered_map<UniqueSimTrackId, TrackingParticleRef, UniqueSimTrackIdHash>>"/>
   <class name="SimTrackToTPMap"/>
   <class name="edm::Wrapper<SimTrackToTPMap>"/>
+
+  <class name="SimDoublets"/>
```

Comment:
This would need a class version and checksum (starting with version 3).

Or alternatively the `Wrapper<vector<SimDoublets>>` would have the declared as transient.